Sources/StreamChatUI/CommonViews/QuotedChatMessageView/QuotedChatMessageView.swift

```diff
@@ -225,30 +234,31 @@ open class QuotedChatMessageView: _View, ThemeProvider, SwiftUIRepresentable {
     /// the attachments preview of the message, or if you want to support your custom attachment.
     /// - Parameter message: The message that contains all the attachments.
     open func setAttachmentPreview(for message: ChatMessage) {
+        let currentMessageText = textView.text
         if let filePayload = message.fileAttachments.first?.payload {
             attachmentPreviewView.contentMode = .scaleAspectFit
             attachmentPreviewView.image = appearance.images.fileIcons[filePayload.file.type] ?? appearance.images.fileFallback
-            textView.text = message.text.isEmpty ? filePayload.title : message.text
+            textView.text = message.text.isEmpty ? filePayload.title : currentMessageText
         } else if let imagePayload = message.imageAttachments.first?.payload {
             attachmentPreviewView.contentMode = .scaleAspectFill
             setAttachmentPreviewImage(url: imagePayload.imageURL)
-            textView.text = message.text.isEmpty ? "Photo" : message.text
+            textView.text = message.text.isEmpty ? "Photo" : currentMessageText
```

Comment:
Didn't we localize these at some point? Or was that a different part?

Reply:
This is a different part, this one is not localized yet. Currently it is out of scope of this PR